features/step_definitions/action_page_steps.rb

```diff
@@ -4,18 +4,41 @@
 end
 
 When(/^I browse to an embedded action targetting Nancy Pelosi and Kamala Harris$/) do
-  params = "?bioguide_ids[]=P000197&bioguide_ids[]=H001075"
-  visit "/action/#{@action_page.title.downcase.gsub(" ", "-")}/embed_iframe#{params}"
+  visit "/action/#{@action_page.title.downcase.gsub(" ", "-")}/embed_iframe?" + {
+    bioguide_ids: ["P000197", "H001075"],
+    zip4: "7701",
+    zipcode: "94109",
+    street: "815 Eddy Street",
+    city: "San Francisco",
+    state: "CA"
+  }.to_param
 end
 
-When(/^I fill out the required fields for Nancy Pelosi$/) do
+When(/^I fill in the required fields for Nancy Pelosi$/) do
   select("Ms.", from: "$NAME_PREFIX")
   fill_in("$NAME_FIRST", with: "Buffy")
   fill_in("$NAME_LAST", with: "Summers")
   fill_in("$EMAIL", with: "bsummers@ucsunnydale.edu")
   select("Congress", from: "$TOPIC")
 end
 
+When(/^I fill in the required fields for Nancy Pelosi and Kamala Harris$/) do
```

Comment:
A few tests are failing because they use "I fill *out* the required fields..." for these steps.